### PR TITLE
fix: Legacy CI `pip` installation

### PR DESCRIPTION
**Pull request by gs-olive**
- Update `config.yaml` to support legacy CI testing, including upgraded `.toml` and `WORKSPACE` files built specifically for the legacy version
- Add parameters to the configuration file to allow switching CUDA versions for the legacy build
- Update default in `compile()` to be TorchScript in the legacy install

### CODE REVIEW

**Comment by gs-olive** [comment]
toolchains special toml for legacy

File: py/torch_tensorrt/_compile.py [hunk 1]
```diff
@@ -93,13 +100,14 @@ def _get_target_ir(module_type: _ModuleType, ir: str) -> _IRType:
     else:
         if ir == "default":
             # Options are listed in order of preference
-            if module_is_fxable:
+            if DYNAMO_ENABLED and module_is_fxable:
```
Comment (gs-olive) [inline py/torch_tensorrt/_compile.py:103]: Update default path to be TorchScript if the install is with a legacy Torch version

**Review by narendasan — APPROVED** [review]
LGTM

**Review by peri044 — APPROVED** [review]
Looks great

**Pull request merged**